sdks/java/io/redis/build.gradle

```diff
@@ -28,7 +28,7 @@ dependencies {
   implementation "redis.clients:jedis:4.0.1"
   testImplementation project(path: ":sdks:java:io:common", configuration: "testRuntimeMigration")
   testImplementation library.java.junit
-  testImplementation "io.oden:embedded-redis:0.0.3"
+  testImplementation "org.signal:embedded-redis:0.8.2"
```

Comment:
Please add the following to `build.gradle` (at the very end is fine) to allow the required version of commons-io for test runs.

```
configurations.testRuntimeClasspath {
  // force version required by embedded-redis:0.8.2 in tests
  resolutionStrategy { force "commons-io:commons-io:2.7" }
}
```

Reply:
Thank you, I was looking at the test failures and absolutely losing my mind trying to figure out what was going on.